packages/next/src/server/app-render/app-render.tsx

```diff
@@ -73,6 +73,8 @@ import { appendMutableCookies } from '../web/spec-extension/adapters/request-coo
 import { createServerInsertedHTML } from './server-inserted-html'
 import { getRequiredScripts } from './required-scripts'
 import { addPathPrefix } from '../../shared/lib/router/utils/add-path-prefix'
+import { StaticGenerationStore } from '../../client/components/static-generation-async-storage.external'
```

Comment:
We should `import type` here so they can easily be removed by swc transform

Reply:
opened a PR to help detect this in the future since it has come up in the past: https://github.com/vercel/next.js/pull/56551